HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -847,11 +847,16 @@ def self.apply_dehumidifiers(runner, model, dehumidifiers, conditioned_space, un
   def self.apply_ceiling_fans(model, runner, weather, ceiling_fan, conditioned_space, schedules_file,
                               unavailable_periods)
     obj_name = Constants.ObjectNameCeilingFan
-    medium_cfm = 3000.0 # From ANSI 301-2019
     hrs_per_day = 10.5 # From ANSI 301-2019
     cfm_per_w = ceiling_fan.efficiency
+    label_energy_use = ceiling_fan.label_energy_use
     count = ceiling_fan.count
-    annual_kwh = UnitConversions.convert(count * medium_cfm / cfm_per_w * hrs_per_day * 365.0, 'Wh', 'kWh')
+    if !label_energy_use.nil? # priority if both provided
+      annual_kwh = UnitConversions.convert(count * label_energy_use * hrs_per_day * 365.0, 'Wh', 'kWh')
+    elsif !cfm_per_w.nil?
+      medium_cfm = get_default_ceiling_fan_medium_cfm()
+      annual_kwh = UnitConversions.convert(count * medium_cfm / cfm_per_w * hrs_per_day * 365.0, 'Wh', 'kWh')
+    end
```

Comment:
I was surprised to find that we don't have any ceiling fan unit tests in `test_lighting.rb`; could you add some? It'd be nice to have a test for the `CeilingFanSetpointTempCoolingSeasonOffset` input too, I guess that would be a setpoint-related test that should live in `test_hvac.rb`.